src/app/space/create/deployments/apps/deployment-card.component.html

```diff
@@ -38,8 +38,8 @@
           </div>
         </div>
       </div>
-      <deployment-details [collapsed]="collapsed" [applicationId]="applicationId"
-        [environment]="environment" [spaceId]="spaceId"></deployment-details>
+      <deployment-details onclick="event.preventDefault(); event.stopPropagation();" [collapsed]="collapsed"
+        [applicationId]="applicationId" [environment]="environment" [spaceId]="spaceId"></deployment-details>
```

Comment:
Some other component views [1] also stop propagation but only calling `event.stopPropagation()`. What does `preventDefault` do? Is it necessary elsewhere as well?

[1] `deployment-card.component.html` and `deployments-donut.component.html`

Reply:
I just tested it quickly and `stopPropagation()` on its own seems sufficient, but I don't know what's best practice here really. I was hoping Adam might have some input here.

There's this discussion that has quite a bit of info: https://stackoverflow.com/questions/5963669/whats-the-difference-between-event-stoppropagation-and-event-preventdefault

But I still don't have a clear picture...

Reply:
from an interaction perspective, if this card is clickable, there should be a cursor change and some kind of highlight on the card when the user hovers over it.

The Single-Select Card View demo has the best example: http://www.patternfly.org/pattern-library/content-views/card-view/. The classes are included with the framework.

Reply:
The cursor change and highlight are good things to have, but I think it would fall into a separate issue and PR.

I think if only `stopPropagation()` is sufficient, we should leave it at that for now.

Reply:
I'm good with the style pieces being a separate PR/issue.